plasmapy/analysis/tests/test_fit_functions.py

```diff
@@ -9,7 +9,7 @@
 
 import plasmapy.analysis.fit_functions as ffuncs
 
-# ^ ExitStack can be replaced with nullcontext when we require >= python 3.7
+# TODO: replace ExitStack with nullcontext when we require >= python 3.7
```

Comment:
We should be able to do this now, or at least soon!

Reply:
Done!